metaflow/plugins/kfp/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .kfp_utils import (
+    get_kfp_run,
+    get_pipeline_versions,
+    get_pipeline_versions_by_id,
+    is_finished_run,
+    is_successful_run,
+    logger,
+    run_id_to_url,
+    run_kubeflow_pipeline,
+    run_kubeflow_pipeline_by_id,
+    terminate_run,
+    wait_for_kfp_run_completion,
```

Comment:
Let's limit the exposure and SDK set to the Applied Scientist.

```python
run_kubeflow_pipeline()
is_finished_run() # if the above function has the word "pipeline" so should this one, and to not confuse with Metaflow run
is_successful_run()
```

I don't see the need of the other functions...

Reply:
Can you please add an example Flow in https://github.com/zillow/metaflow/tree/feature/kfp/metaflow/tutorials/09-hellokfp of a Flow calling another Flow?  This would help us determine exactly what functions are needed, thanks!